engines/mohawk/myst.cpp

```diff
@@ -1161,6 +1164,12 @@ Common::Error MohawkEngine_Myst::saveGameState(int slot, const Common::String &d
 	return _gameState->save(slot, desc) ? Common::kNoError : Common::kUnknownError;
 }
 
+void MohawkEngine_Myst::autoSave() {
+	int autoSaveSlot = 0;
```

Comment:
This probably should be a constant so it can be used in other places when necessary.